Sources/ComposableArchitecture/Store.swift

```diff
@@ -179,7 +181,8 @@ public final class Store<State, Action> {
       initialState: initialState,
       reducer: reducer,
       environment: environment,
-      mainThreadChecksEnabled: false
+      mainThreadChecksEnabled: false,
+      stackChecksEnabled: true
```

Comment:
Because this initializer is mostly used for background stores for which the stack size is smaller, we probably want stack checks to be enabled.

Reply:
I finally switched it back to `false` in order to be coherent with "unchecked".